composer.json

```diff
@@ -6,10 +6,13 @@
         "vendor-dir": "Packages/Libraries",
         "bin-dir": "bin",
         "allow-plugins": {
-            "neos/composer-plugin": true
+            "neos/composer-plugin": true,
+            "ocramius/package-versions": true
         }
     },
     "require": {
+        "php": "^8.1",
```

Comment:
Doh! This was actually the first thing I did, and then removed it again because I decided to do a patch-level update before anything bigger… Thanks!